docs/topics/cache.txt

```diff
@@ -169,6 +169,121 @@ particularly temporary.
     some problems and seems to be unmaintained. Use ``PyMemcacheCache`` or
     ``PyLibMCCache`` instead.
 
+.. _redis:
+
+Redis
+-----
+
+Redis__ is an in-memory database that can be used for caching. To begin you'll
+need a Redis server running either locally or on a remote machine.
+
+__ https://redis.io/
+
+After setting up the Redis server, you'll need to install Python bindings for
+Redis. `redis-py`_ is the binding supported natively by Django.
+
+.. _`redis-py`: https://pypi.org/project/redis/
+
+To use Redis as your cache backend with Django:
+
+* Set :setting:`BACKEND <CACHES-BACKEND>` to
+  ``django.core.cache.backends.redis.RedisCache``
+
+* Set :setting:`LOCATION <CACHES-LOCATION>` to the URL pointing to your Redis 
+  instance, using the appropriate scheme. See the example below, and the 
+  ``redis-py`` docs for `details on the available schemes 
+  <https://redis-py.readthedocs.io/en/stable/#redis.ConnectionPool.from_url>`_. 
+
+For example, if Redis is running on localhost (127.0.0.1) port 6379::
+
+    CACHES = {
+        'default': {
+            'BACKEND': 'django.core.cache.backends.redis.RedisCache',
+            'LOCATION': 'redis://127.0.0.1:6379',
+        }
+    }
+
+Often Redis servers are protected with authentication. In order to supply a
+username and password, add them in the ``LOCATION`` along with the URL::
+
+    CACHES = {
+        'default': {
+            'BACKEND': 'django.core.cache.backends.redis.RedisCache',
+            'LOCATION': 'rediss://username:password@127.0.0.1:6379',
```

Comment:
```suggestion
            'LOCATION': 'redis://username:password@127.0.0.1:6379',
```
....ssss :snake:

Reply:
Actually, this isn't incorrect. `rediss://` is for Redis-over-TLS. See the following:

- https://github.com/jazzband/django-redis#configure-as-cache-backend
- https://www.iana.org/assignments/uri-schemes/prov/rediss

But it does look like a typo...

Reply:
No I actually did this on purpose 😛 
Since we had examples of `redis://` and `unix://` for the `LOCATION`, I though of using `rediss://` for this one!

Reply:
> No I actually did this on purpose 

I get that. I meant that the choice of `rediss` as the scheme name, while it follows `http` → `https`, just makes it look like a typo when it isn't.

Reply:
I don't think that we need examples for all available options. Switching between `redis` and `rediss` without any comment, only to put all available schemas in the docs is confusing. I'd use `redis` instead.